config/environments/sit.rb

```diff
@@ -99,6 +99,7 @@
   config.cmr_env = 'sit'
   config.echo_env = 'sit'
   config.analytics_id = 'UA-62340125-5'
+  config.tag_manager_id = 'GTM-TESTTEST'
```

Comment:
what was the logic for not using the ID for SIT and UAT? I see why to not use it for development and test. Not yet suggesting a change, just want to ask what lead us to here.

Reply:
Peggy from EMS replied that same ID can be used in all environments.  Updated code & resubmitted.